test_runner/regress/test_import_pgdata.py

```diff
@@ -332,6 +340,224 @@ def validate_vanilla_equivalence(ep):
         br_initdb_endpoint.safe_psql("select * from othertable")
 
 
+def test_fast_import_with_pageserver_ingest(
+    test_output_dir,
+    vanilla_pg: VanillaPostgres,
+    port_distributor: PortDistributor,
+    fast_import: FastImport,
+    pg_distrib_dir: Path,
+    pg_version: PgVersion,
+    mock_s3_server: MockS3Server,
+    mock_kms: KMSClient,
+    mock_s3_client: S3Client,
+    neon_env_builder: NeonEnvBuilder,
+    make_httpserver: HTTPServer,
+):
```

Comment:
It is already reviewed and approved https://github.com/neondatabase/neon/pull/10487

Reply:
Should we rebase then (or merge w main)? I don't get why it's not conflicting.